schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,40 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Template of a remote configuration for the Rapid Recommendations feature of the Insights Operator.",
+  "example": {
+    "conditional_gathering_rules": [
+      "conditional_gathering_rules/*.json"
+    ],
+    "container_logs": [
+      "container_log_requests/ccx-rules-ocp/*.json",
+      "container_log_requests/manual/kube_controller_manager_logs_gatherer.json"
+    ]
+  },
+  "properties": {
+    "conditional_gathering_rules": {
+      "description": "Array of glob patterns matching conditional gathering rules to be included in the configuration file. The glob patterns are relative to the source data (repository) root.",
+      "items": {
+        "description": "A glob pattern selecting conditional gathering rules",
+        "pattern": "^conditional_gathering_rules/[^/]+\\.json$",
+        "type": "string"
+      },
+      "type": "array"
+    },
+    "container_logs": {
+      "description": "Array of glob patterns matching conditional gathering rules to be included in the configuration file. The glob patterns are relative to the source data (repository) root.",
```

Sorry, this might have been a typo in my suggestion:

```suggestion
      "description": "Array of glob patterns matching container log requests to be included in the configuration file. The glob patterns are relative to the source data (repository) root.",
```